nix/build-all.py

```diff
@@ -262,27 +267,40 @@ def v(dep):
         raise ValueError("Required tool '%s' not installed or not added to PATH" % (cmd,))
 
 # identifiers for the download tool (could be less memory consuming as ints, but are more verbose as strings)
-download_tool_default = download_tool_py = "py"
+download_tool_curl="curl"
+download_tool_wget="wget"
 download_tool_git = "git"
 
+if which(wget) != None:
```

Comment:
Is there a reason why you re-enabled this related to wget and curl? Or was it a merge error?

Reply:
I didn't touch this part specifically. But, the code looks ok, right?

Reply:
> I didn't touch this part specifically.

Then it was probably a merge error.

> But, the code looks ok, right?

These bits of code were removed recently (on Jun 30: https://github.com/IfcOpenShell/IfcOpenShell/commit/51cf52c38848534ac3ef5210d97a77cb8fe22819) wget was giving issues on older distros due to all the changes in certificates and TLS recently. But now you're adding them back in your pull request.

I hope you can review the changes in your PR and remove the unintentional and whitespace changes. If you have time for making your change conditional that would be great as well. Let me know, I can also assist.